org.civicrm.bootstrapcivihr/scss/directives/horizontal-scroll-shadows.scss

```diff
@@ -0,0 +1,40 @@
+.horizontal-scroll-shadows-master {
```

Comment:
there shouldn't be a "directives/" folder, the style shouldn't really care whether the classes come from a directive, a jquery plugin, or whatever

Reply:
@AkA84 where would you like me to put this file then?

Reply:
write it as a BEM component, put it in components/

Reply:
I have to put it in components out of namespace

Reply:
I had to put it in components/outside-namespace, otherwise, won't work for SSP